config/crd/bases/metrics.console.redhat.com_floorplans.yaml

```diff
@@ -32,7 +32,29 @@ spec:
           spec:
             description: Spec defines the desired state of FloorPlan
             type: object
-            x-kubernetes-preserve-unknown-fields: true
+            required:
+              - queries
+            properties:
```

Comment:
Needs more properties, like db name and bucket name.
Check https://github.com/RedHatInsights/floorist/blob/master/clowdapp.yaml